docs/web-ui.md

```diff
@@ -363,6 +363,42 @@ number of written shuffle records, total data size, etc.
 Clicking the 'Details' link on the bottom displays the logical plans and the physical plan, which
 illustrate how Spark parses, analyzes, optimizes and performs the query.
 
+### SQL metrics
+
+The metrics of SQL operators show in the block of operators. The SQL metrics can be useful when
```

Comment:
@viirya a little confused on "show in the block of operators" ? Is there a way to reword this ?